#  These controllers are loaded by the robot.launch script, the following types are available
#  type: cardsflow/kindyn/TorquePositionController
#  type: cardsflow/kindyn/CableLengthController
#  type: cardsflow/kindyn/CableLengthVelocityController
#  type: cardsflow/kindyn/ForcePositionController
joint_hip_left:
  joint: joint_hip_left
  type: cardsflow/kindyn/CableLengthController
joint_hip_right:
  joint: joint_hip_right
  type: cardsflow/kindyn/CableLengthController
joint_knee_left:
  joint: joint_knee_left
  type: cardsflow/kindyn/CableLengthController
joint_knee_right:
  joint: joint_knee_right
  type: cardsflow/kindyn/CableLengthController
joint_foot_left:
  joint: joint_foot_left
  type: cardsflow/kindyn/CableLengthController
joint_foot_right:
  joint: joint_foot_right
  type: cardsflow/kindyn/CableLengthController

left_shoulder_axis0:
  joint: left_shoulder_axis0
  type: cardsflow/kindyn/CableLengthController
left_shoulder_axis1:
  joint: left_shoulder_axis1
  type: cardsflow/kindyn/CableLengthController
left_shoulder_axis2:
  joint: left_shoulder_axis2
  type: cardsflow/kindyn/CableLengthController
elbow_left_rot0:
  joint: elbow_left_rot0
  type: cardsflow/kindyn/CableLengthController
elbow_left_rot1:
  joint: elbow_left_rot1
  type: cardsflow/kindyn/CableLengthController
left_wrist_0:
  joint: left_wrist_0
  type: cardsflow/kindyn/CableLengthController
left_wrist_1:
  joint: left_wrist_1
  type: cardsflow/kindyn/CableLengthController

right_shoulder_axis0:
  joint: right_shoulder_axis0
  type: cardsflow/kindyn/CableLengthController
right_shoulder_axis1:
  joint: right_shoulder_axis1
  type: cardsflow/kindyn/CableLengthController
right_shoulder_axis2:
  joint: right_shoulder_axis2
  type: cardsflow/kindyn/CableLengthController
elbow_right_rot0:
  joint: elbow_right_rot0
  type: cardsflow/kindyn/CableLengthController
elbow_right_rot1:
  joint: elbow_right_rot1
  type: cardsflow/kindyn/CableLengthController
right_wrist_0:
  joint: right_wrist_0
  type: cardsflow/kindyn/CableLengthController
right_wrist_1:
  joint: right_wrist_1
  type: cardsflow/kindyn/CableLengthController

spine_joint:
  joint: spine_joint
  type: cardsflow/kindyn/CableLengthController
head_axis0:
  joint: head_axis0
  type: cardsflow/kindyn/CableLengthController
head_axis1:
  joint: head_axis1
  type: cardsflow/kindyn/CableLengthController
head_axis2:
  joint: head_axis2
  type: cardsflow/kindyn/CableLengthController

joint_wheel_right:
  joint: joint_wheel_right
  type: cardsflow/kindyn/TorquePositionController
joint_wheel_back:
  joint: joint_wheel_back
  type: cardsflow/kindyn/TorquePositionController
joint_pedal:
  joint: joint_pedal
  type: cardsflow/kindyn/TorquePositionController
joint_wheel_left:
  joint: joint_wheel_left
  type: cardsflow/kindyn/TorquePositionController
joint_front:
  joint: joint_front
  type: cardsflow/kindyn/CableLengthController
joint_pedal_right:
  joint: joint_pedal_right
  type: cardsflow/kindyn/TorquePositionController
joint_pedal_left:
  joint: joint_pedal_left
  type: cardsflow/kindyn/TorquePositionController